### ISSUE TITLE
more details about hyper-parameter settings?  

### ISSUE THREAD

**Issue by zengyan-97** [clovaai/som-dst#1]
Hi, 
Thanks for sharing the code! Could you also share the hyper-parameter settings to reproduce  53.09% joint accuracy on MultiWOZ 2.1? 

I cannot get this result by directly running the code (without modification) parallelly on two P100 GPUs. I get 52.16% at the 29th epoch  (best on dev set).  

I also found that a paper ("Efficient Context and Schema Fusion Networks for Multi-Domain
Dialogue State Tracking") reported that your model achieves only 52.57%, which result is now shown at paperswithcode.

**Comment by DSKSD** [comment]
@zengyan-97 
Sorry for the late reply..!
The `52.57` is the previous result (arxiv version).
And we found there is non-deterministic function `scatter_add`.
(Please see, https://pytorch.org/docs/1.4.0/notes/randomness.html)
So, we report the result `53.01`  by averaging 10 results
